### PR TITLE
Aside page navigation

### PR DESCRIPTION
**Pull request by nickhsine**
@garfieldduck @hcchien

### CODE REVIEW

File: src/components/article/PageNavigation.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+'use strict'
+import { Link } from 'react-router'
+import _ from 'lodash'
+import classNames from 'classnames'
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+import styles from './PageNavigation.scss'
```
Comment (hcchien) [inline src/components/article/PageNavigation.js]: sorting? (upper case, lower case)

**Comment by hcchien** [comment]
👍

File: src/containers/Article.js [hunk 2]
```diff
@@ -188,7 +188,14 @@ export default class Article extends Component {
...
         </div>
-
+        <ArticleComponents.PageNavigation
+          article={ _.get(article, [ 'relateds', 0 ])}
+          navigate="next"
+        />
+        <ArticleComponents.PageNavigation
+          article={_.get(article, [ 'relateds', 1 ])}
+          navigate="previous"
+        />
```
Comment (garfieldduck) [inline src/containers/Article.js]: Do we need further discussion on how to let users go back to the previous article they've just clicked? Sry I just found NYT can let user go back while the first previous article might be selected from its backend

Comment (nickhsine) [inline src/containers/Article.js]: Sorry, it's just a temporary solution for showing next and previous articles.
It should be discussed later after the designer and HC come into office.

**Comment by garfieldduck** [comment]
LGTM 👍 👍 👍

**Pull request merged**